src/patent_fetcher.py

```diff
@@ -28,9 +28,14 @@ def conduct_search(title: str, author=False, status=False, expiry=False, lim=100
 
 
 @celery.task(name="download_patents")
-def download_patents(patent_list: List[USApplication], destination: str):
+def download_patents(destination: str,
+                     patent_list: List[USApplication] = None,
+                     patent_title: str = None):
     """ Download all the patent drawings that arose from the search to the downloads/folder"""
-    if destination[-1] != "/":
+    if patent_list is None and patent_title is not None:
```

Comment:
hence as a workaround I do the search here, but now that happens twice != ideal!